forge-gui/res/adventure/common/decks/standard/peasantrole.dck

```diff
@@ -0,0 +1,43 @@
+[metadata]
+Name=peasantrole
+[Avatar]
+
+[Main]
+2 Ambitious Farmhand|MID|1
+1 Ellivere of the Wild Court|WOC|1
+1 Farmer Cotton|LTC|1
+4 Forest|WOE|1
+1 Forest|WOE|2
+6 Forest|WOE|3
+1 Giant Inheritance|WOC|1
+3 Giant Killer|PLIST|1
+1 Gylwain, Casting Director|WOC|1
+2 Hopeful Eidolon|THS|1
+2 Hyena Umbra|UMA|1
+1 Lurrus of the Dream-Den|Q06|1
+1 Ossification|ONE|1
+2 Pearl-Ear, Imperial Advisor|MH3|1
+7 Plains|WOE|1
+1 Plains|WOE|2
+4 Plains|WOE|3
+2 Redtooth Genealogist|WOE|1
+1 Return Triumphant|WOE|1
+3 Royal Treatment|WOE|1
+1 Songbirds' Blessing|WOC|1
+3 Spellbook Vendor|WOE|1
+2 Spider Umbra|UMA|1
+2 Springheart Nantuko|MH3|1
+3 Stroke of Midnight|WOE|1
+1 Sun Titan|ZNC|1
+1 Syr Armont, the Redeemer|WOE|1
+1 Well Rested|PIP|2
+[Sideboard]
+
+[Planes]
+
+[Schemes]
+
+[Conspiracy]
+
+[Dungeon]
```

Comment:
Some decks like this one have empty definitions for sections, can prolly be cleaned up later if needed, though functionally it doesn't affect anything.